templates/telco/index.html

```diff
@@ -1,42 +1,61 @@
 {% extends "telco/base_telco.html" %}
 
-{% block title %}Open source NFV platform for 5G{% endblock %}
+{% block title %}Carrier-grade open source for telecommunications{% endblock %}
 
-{% block meta_description %}Ubuntu is a leading platform in the telecommunications sector. Ubuntu powers telecommunications infrastructure of leading global service providers. Ubuntu is in the heart of the entire ETSI NFV stack, providing both NFVI and MANO, bringing operational costs down and ensuring the highest level of security. All of that at a competitive price.{% endblock %}
+{% block meta_description %}Canonical in telecommunications - secure open source telecom solutions for private mobile networks, Open RAN deployments, edge computing use cases, and private and edge cloud infrastructure, consumable by mobile network operators and enterprises.{% endblock %}
 
 {% block meta_copydoc %}https://docs.google.com/document/d/1a3otoXCbXtddJG2HCnYIIHJiDxvG-pzFPtLobVB4-vw/edit{% endblock meta_copydoc %}
 
 {% block content %}
 <section class="p-strip--suru is-dark">
-  {% include "shared/_rt-kernel-ga-banner.html" %}
-
   <div class="row u-equal-height">
-    <div class="col-8 u-vertically-center">
-      <h1>
-        Open source for telecoms
-      </h1>
-      <p class="p-heading--3">
-        Carrier-grade solutions
-      </p>
-      <p class="u-sv3">
-      Scale and complexity brings similar challenges, big or small. Hybrid cloud, public cloud, on-prem only large datacenter or highly distributed infrastructure - Canonical Open Source technology building blocks provide a unified approach to enabling Service Providers to meet any current or future use-cases, from OpenRAN, next generation Core (5G and beyond) or AI at the edge.
-      </p>
+    <div class="col-7 u-vertically-center">
+      <h1>Carrier-grade open source for telecom</h1>
+      <p class="p-heading--3">Canonical offers secure, modular and interoperable solutions for public and private clouds, multi-cloud or hybrid-cloud environments at centralised or distributed sites.</p>
       <p>
-        <a href="/telco/contact-us" class="p-button--positive js-invoke-modal">Contact us</a>
+        <a href="/telco/contact-us" class="p-button--positive">Contact us</a>
```

Comment:
The CTA at the bottom is linking to the modal, it looks a bit inconsistent that this one doesn't open the modal?

Reply:
Had another look at the doc and it's actually not supposed to link to the modal at the bottom either, so I've updated